pipelines/environments/mldesigner-2.0-ubuntu20.04-py38-cpu/asset.yaml

```diff
@@ -0,0 +1,4 @@
+name: mldesigner-2.0-ubuntu20.04-py38-cpu
```

Comment:
Do you plan to use automated versioning? If so, you should include `version: auto` See https://github.com/Azure/azureml-assets/wiki/Environments#assetyaml

Reply:
Yes, and added `version: auto` for this, thanks for pointing out!